app/scripts/modules/amazon/src/vpc/VpcTag.tsx

```diff
@@ -20,20 +20,20 @@ export class VpcTag extends React.Component<IVpcTagProps, IVpcTagState> {
   }
 
   private updateState(props: IVpcTagProps): void {
```

Comment:
fwiw, ideally the initial call to `updateState` should be in a `componentDidMount`, not the constructor.